src/clients/node/docs.zig

```diff
@@ -87,7 +87,7 @@ pub const NodeDocs = Docs{
     \\console.log("Import ok!");
     ,
 
-    .install_commands = "npm install tigerbeetle-node",
+    .install_commands = "npm install --prefix ../../ && npm install",
```

Comment:
It makes the instruction Windows friendly:

```suggestion
    .install_commands = 
    \\npm install --prefix ../../
    \\npm install
    ,
```